frontend/src/types/common.ts

```diff
@@ -14,6 +14,7 @@ export const TEXT_VALIDATION_DELAY = 1000;
 export const ENTERPRISE_INQUIRE_LINK =
   "https://www.bytebase.com/contact-us?source=console";
 
+export const OPENAI_DEFAULT_MODEL = "gpt-3.5-turbo";
```

Comment:
We want to limit the model's input to predefined options rather than allowing any arbitrary string.

Reply:
Sometimes, users need to use their own self-built large model services. For example, like me. These services are compatible with the OpenAI interface. However, the names of the models may all be custom.

Reply:
Alright, let's move forward for now, and we can enhance the setup experience later.